urdf/ergoCub/conf/gazebo_ergocub_left_hand.ini

```diff
@@ -3,14 +3,14 @@ yarpDeviceName left_hand_hardware_device
 jointNames l_thumb_add l_thumb_prox l_thumb_dist l_index_add l_index_prox l_index_dist l_middle_prox l_middle_dist l_ring_prox l_ring_dist l_pinkie_prox l_pinkie_dist
 
 [COUPLING]
-icub_hand_mk5 (0 1 2 3 4 5 6 7 8 9 10 11) (l_thumb_add l_thumb_oc l_index_add l_index_oc l_middle_oc l_ring_pinky_oc reserved reserved reserved reserved reserved reserved)
+icub_hand_mk5 (0 1 2 3 4 5 6 7 8 9 10 11) (l_thumb_add l_thumb_prox l_thumb_dist l_index_add l_index_prox l_index_dist l_middle_prox l_middle_dist l_ring_prox l_ring_dist l_pinkie_prox l_pinkie_dist)
```

Comment:
This change is reverting the refactoring we did #54, please remove it.

Moreover, the last joints have to remain `reserved` for the coupling laws